server/server/models/users.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship
+
+
+class User:
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    email = Column(String, unique=True)
+    nickname = Column(String, enique=True)
+    password = Column(String)
+    create_date = Column(String)
+    location = Column(String)
+    f_name = Column(String)
+    l_name = Column(String)
```

Comment:
I have no idea what `f_name` and `l_name` mean. It's better to use some self-explaining names.